submissions/onyinyeChukwuka/HTML-Forms/contactform.html

```diff
@@ -0,0 +1,32 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>CONTACT FORM</title>
+</head>
+<body>
+    <form action="" method="POST"> 
+        <fieldset>
+            <legend><h4>CONTACT US</h4></legend>
+            <!-- This is for the User's fullname-->
+            <label for="fname">Full Name:</label><br>
+            <input type="text" id="fullname" name="fullname"><br><br>
+
+            <!-- This is for the User's Email-->
+            <label for="fname">Email:</label><br>
```

Comment:
update the label for the email @Onyinye101

Reply:
I've made the corrections